### DISCUSSION TITLE
Combine two edited videos

### DISCUSSION THREAD

**Discussion by AndiLeni** [mifi/lossless-cut#2122] [Support] [answered]
Hello,

I am unsure how to handle this case propery and could not find any guides online.

I have two different video files, where the ending of the movie from the first file is in the second file.

The files are edited like this:

File 1:
![image](https://github.com/user-attachments/assets/b8180e86-d167-41d8-b281-0b4d07f272fd)

File 2:
![image](https://github.com/user-attachments/assets/4264998c-c83c-4af4-89ac-81595d46909e)

What I need is to combine the two parts from each movie to a single video file.
Exporting each video and then combining them afterwards feels wrong.
What is the correct approach to achieve this?

thanks and kind regards

**Comment by mifi** [comment, marked as answer]
that sounds to me like the correct way of doing it. why does it feel wrong?

**Reply by AndiLeni** [reply]
Depending on the number of videos to combine, it creates many, possibly large, files which are combined afterwards and can then get deleted. 

I just thought there might be a way to skip this step in between to save the videos each and just create a single combined output file. 
This also saves a manual step :)

**Reply by mifi** [reply]
ah - that sounds like non-linear editing, which losslesscut does not (and will probably not) support #976

**Reply by AndiLeni** [reply]
That's what I thought. I have now done it as you described. Thanks again.
